cmd/validatorcmd/getBalance.go

```diff
@@ -58,68 +71,137 @@ func getBalance(_ *cobra.Command, _ []string) error {
 		return err
 	}
 
-	var balance uint64
-	if validationIDStr != "" {
-		validationID, err := ids.FromString(validationIDStr)
-		if err != nil {
-			return err
-		}
-		balance, err = txutils.GetValidatorPChainBalanceValidationID(network, validationID)
-		if err != nil {
-			return err
-		}
-		ux.Logger.PrintToUser("  Validator Balance: %.5f", float64(balance)/float64(units.Avax))
+	validationID, cancel, err := getNodeValidationID(network, l1, nodeIDStr, validationIDStr)
+	if err != nil {
+		return err
+	}
+	if cancel {
 		return nil
 	}
+	if validationID == ids.Empty {
+		return fmt.Errorf("the specified node is not a L1 validator")
```

Comment:
nit: add `as validationID is empty` to the error msg
pls feel free to ignore this

Reply:
ignoring as in the other PR